### PR TITLE
added base components for react

### LINKED ISSUE #57: Front end framework opstellen
What has to be done:
Learn React.
Add Routing.
Create all components which can be routed to.
Show mock-data in each component.

Why it has to be done:
So that a working structure is created that everyone can adhere to.

When the task is done:
We can start to show real data and beautify the components.

### PR DESCRIPTION
**Pull request by MichielLangen**
closes #57 

Made by Geert, due to problems with pushing I created the pull request.

During stand-up we agreed not all the components had to be made, this is to help the others get started

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -2,23 +2,27 @@ import React, { Component } from "react"
 import "./App.css"
 import firebase from "firebase"
 import StyledFirebaseAuth from "react-firebaseui/StyledFirebaseAuth"
+import Navigator from './components/routerComponent';
+
+// Your web app's Firebase configuration
+var firebaseConfig = {
```
Comment (chf3kl9) [inline src/App.js]: I think we should hide/protect this information, as it now is made public (which is something we don't want)

File: src/components/usersComponent.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
+import React, { Component } from "react";
+
+import Userdetail from "./userdetailComponent";
+
+class Users extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
```
Comment (chf3kl9) [inline src/components/usersComponent.js]: Maybe we can do this outside of the constructor?
We should decide whether we want to do this everywhere in the constructor or outside the constructor for consistency.

Comment (MichielLangen) [reply src/components/usersComponent.js]: Inside the constructor is most commonly used, I suggest we do the same

**Comment by MichielLangen** [comment]
Tried using GitHub secrets, but could not get it to work
Maybe someone else will have more succes

File: src/App.js (hunk 1 above)
Comment (MichielLangen) [reply src/App.js]: Tried using GitHub secrets, but could not get it to work
Maybe someone else will have more succes

Comment (MichielLangen) [reply src/App.js]: "***The apiKey essentially just identifies your Firebase project on the Google servers. It is not a security risk for someone to know it. In fact, it is necessary for them to know it, in order for them to interact with your Firebase project.***"

So this doesn't matter

https://stackoverflow.com/questions/37482366/is-it-safe-to-expose-firebase-apikey-to-the-public

**Pull request merged**